src/Aspire.Dashboard/Components/Pages/Login.razor

```diff
@@ -0,0 +1,49 @@
+﻿@page "/login"
+@using Aspire.Dashboard.Utils
+@layout EmptyLayout
+@attribute [AllowAnonymous]
+@inject IStringLocalizer<Dashboard.Resources.Login> Loc
+
+<PageTitle><ApplicationName ResourceName="@nameof(Dashboard.Resources.Login.PageTitle)" Loc="@Loc" /></PageTitle>
+
+<div class="token-backdrop">
+    <EditForm EditContext="@EditContext" OnValidSubmit="@SubmitAsync" FormName="token_validation">
+        <DataAnnotationsValidator />
+        <div class="token-form-container">
+            <div class="token-logo">
+                <AspireLogo Height="128" Width="128" />
```

Comment:
Safari has issues with displaying our logo svg as a downloaded file. When we moved to embedding it as an icon like this for the header, it fixed the issue. So I stuck with it. It's actually the same size (svg content) for the header, which since that's on all requests might be a bigger concern. I'll file an issue to see if there's any way to optimize the size but I know the designers are incredibly busy at the moment so I don't expect it'll be handled by them soon.